.github/workflows/_linux-test.yml

```diff
@@ -84,6 +84,18 @@ jobs:
       - name: Setup Linux
         uses: ./.github/actions/setup-linux
 
+      - name: Get workflow job id
+        id: get-job-id
+        uses: ./.github/actions/get-workflow-job-id
+        if: always()
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Upload log file
```

Comment:
Nit: May be choose a better description for this to make it clear that it's a background process, i.e. Sync log file to S3 in background` or something.  Thoughts?